### DISCUSSION TITLE
Allow Approvals Bypass For Specific Branches

### DISCUSSION THREAD

**Discussion by Rhunter1** [community/community#31411] [Pull Requests]
It would be really nice to set up the ability for certain branches be able to bypass the required approvals. The use case is when using GitFlow after a release is accepted in `main`, you would want to push those changes into the `dev` branch. It would be nice to bypass the approval requirement as it needed approval to get merged into the main branch.

**Comment by gtnsimon** [comment]
Hello,

Is the discussion [How to find "Allow Specified Actors to bypass required pull requests"? #29771](https://github.com/orgs/community/discussions/29771) can help you ?

You can also refers to the GitHub Docs here: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule#creating-a-branch-protection-rule

**Reply by Rhunter1** [reply]
Though this could fix the problem, by having a user that always bypasses the branch protection when a pull request is from main to dev. It would be nice to have the ability to set up branch protections such that pull requests from a specific branch could bypass the review requirement.
